www/pages/_document.tsx

```diff
@@ -3,9 +3,9 @@ import Document, { Html, Head, Main, NextScript } from 'next/document'
 export default class MyDocument extends Document {
   render() {
     return (
-      <Html lang="en" className="dark">
+      <Html lang="en" className="dark overflow-x-hidden">
         <Head></Head>
-        <body>
+        <body className="lg:overflow-auto">
```

Comment:
what are these changes for?

Reply:
For now, on pc, if you reduce the screen width (under the lg breakpoint), you will be in mobile configuration. You can then open the mobile menu, which disables scrolling on the body. But if you enlarge the width of the screen, the mobile menu is hidden but you still can't scroll the body. So with `lg:overflow-auto` we ensure the body is always scrollable on larger screen sizes.

Reply:
Gotcha! makes sense now 😄 

I think a better way to deal with this could be just to make the mobile menu full width, and remove the hidden property for lg screens on it.

something like the below might mean you don't have to put in utility css classes on`<body>`.

```js
// supabase/www/components/Nav/index.tsx

// line 285
 <div className="p-4 md:p-8 h-screen w-screen fixed bg-white transform overflow-y-scroll -inset-y-0 z-50 dark:bg-dark-700"> ```

Reply:
Yes it's better like this, I changed it.